db/wrapper.go

```diff
@@ -500,6 +514,9 @@ func (q *queryWrapper) exec() error {
 	if q.hint != "" {
 		opts.SetHint(q.hint)
 	}
+	if q.maxTime != nil && *q.maxTime > 0 {
```

Comment:
Is this supposed to disallow max time of 0? It seems like a valid input according to the mongo driver.

Reply:
I'm open to setting it unconditionally, and it doesn't look like the driver will reject it, but what does it _mean_ to set a max time of 0?

Reply:
It's not specified what will happen in the docs but it allows you to pass it. If you pass a negative number, it'll fail. I don't really care about zero maxtime either way.

Reply:
> I don't really care about zero maxtime either way.

So are you saying it's good the way it is? Or is your preference to allow anser users to pass a zero maxtime?

Reply:
I don't care either way, so it's fine to leave it the way it currently is.